platforms/gke/base/use-cases/inference-ref-arch/terraform/comfyui/src/Dockerfile

```diff
@@ -76,17 +76,22 @@ ENV COMFYUI_PORT "8188"
 ARG COMFYUI_CUSTOM_NODE_PATH="${COMFY_HOME}/custom_nodes"
 ARG VHS_REPO="https://github.com/Kosinkadink/ComfyUI-VideoHelperSuite"
 ARG VHS_CUSTOM_NODE_DIR_NAME="${COMFYUI_CUSTOM_NODE_PATH}/ComfyUI-VideoHelperSuite"
+ARG CUSTOM_SCRIPTS_REPO="https://github.com/pythongosssss/ComfyUI-Custom-Scripts"
```

Comment:
You should be able to install this using `comfy-cli`

https://github.com/Comfy-Org/comfy-cli?tab=readme-ov-file#managing-custom-nodes